### PR TITLE
add nodepools config variables

### PR DESCRIPTION
**Pull request by hubertklusek**

### CODE REVIEW

File: terraform/example-data.json [hunk 1]
```diff
@@ -4,7 +4,12 @@
         "bucket_name": "s3-bucket-mqperf",
         "mq": "kafka",
         "cluster_name": "kafka01",
-        "nodes_number": "3",
```
Comment (gkocur) [inline terraform/example-data.json:7]: It's a max number of nodes? So maybe `max_nodes_number`?

File: terraform/example-data.json [hunk 2]
```diff
@@ -4,7 +4,12 @@
...
-        "app_image": "softwaremill/mqperf-kafka:latest"
+        "app_image": "softwaremill/mqperf-kafka:latest",
+        "app_nodes_number": "1",
```
Comment (gkocur) [inline terraform/example-data.json]: same as above

File: terraform/live/_envcommon/eks.hcl [hunk 3]
```diff
@@ -16,27 +16,27 @@ inputs = {
   eks_cluster_node_groups = {
     controllers-pool = {
       min_size       = 1
-      max_size       = 3
+      max_size       = get_env("CONTROLLER_NODES_NUMBER")
       desired_size   = 1
```
Comment (gkocur) [inline terraform/live/_envcommon/eks.hcl]: For EKS it's more complicated. It doesn't scale in the same way as GKE, so probably we want to set desired_size same as max_size.

File: terraform/live/_envcommon/gke.hcl [hunk 4]
```diff
@@ -21,23 +21,23 @@ inputs = {
     {
       name         = "controllers-pool"
       disk_size_gb = 50
-      max_count    = 3
+      max_count    = get_env("CONTROLLER_NODES_NUMBER")
       preemptible  = true
```
Comment (gkocur) [inline terraform/live/_envcommon/gke.hcl]: Using preemtible nodes is a good idea, but in some cases not desirable. I'd make it configurable.

**Review by gkocur — CHANGES_REQUESTED** [review]

File: terraform/live/_envcommon/eks.hcl (hunk 3 above)
Comment (hubertklusek) [reply terraform/live/_envcommon/eks.hcl]: done

File: terraform/example-data.json (hunk 1 above)
Comment (hubertklusek) [reply terraform/example-data.json:7]: done

**Review by gkocur — APPROVED** [review]

**Pull request merged**